security_checks.groovy

```diff
@@ -0,0 +1,54 @@
+
+@Library('PipelineUtils@master')
+import dvsa.aws.mot.jenkins.pipeline.common.RepoFunctions
+import dvsa.aws.mot.jenkins.pipeline.common.GlobalValues
+
+// global variables
+RepoFunctions repoFunctionsFactory = new RepoFunctions()
+GlobalValues globalValuesFactory = new GlobalValues()
+Map<String, Map<String, String>> github = globalValuesFactory.GITHUB_REPOS()
+
+String jenkinsBuildLabel = "cvr"
+String branch = "feature-BL-8921-security-checks" // TODO: change to "master" before merging
```

Comment:
This branch will be used for testing. Please do not merge before I change it to master.